grails-app/services/au/org/ala/biocollect/CommentService.groovy

```diff
@@ -16,17 +16,25 @@ class CommentService {
     }
 
     def deleteComment(data){
-        webService.deleteWrapper(grailsApplication.config.ecodata.service.url + "/comment/${data['id']}?userId=${data['userId']}")
+        Boolean alaAdmin = authService.userInRole(grailsApplication.config.security.cas.adminRole)
```

Comment:
Are we supporting FC_ADMIN in BioCollect?  We have these kind of checks in UserService (e.g. isUserAlaOrFCAdmin)   Even if we aren't it's still nice if the UserService is the main place to make these checks / needs to know those config items)